CodeReview/src/items/VReviewComment.cpp

```diff
@@ -36,25 +36,60 @@ namespace CodeReview
 
 DEFINE_ITEM_COMMON(VReviewComment, "item")
 
-VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* node, const StyleType* style)
-	: Super{parent, node, style}
+VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* nodeType, const StyleType* style)
```

Comment:
You shouldn't care about hiding that function, since it returns `node` anyway. So just use `node` in the second line and you're done.